blynklib_mp.py

```diff
@@ -203,6 +218,13 @@ def _get_socket(self):
             self._socket = socket.socket()
             self._socket.connect(socket.getaddrinfo(self.server, self.port)[0][-1])
             self._set_socket_timeout(self.SOCK_TIMEOUT)
+            if self.port == 443 or self.port == 8443:
+                self.log('Using SSL socket...')
+                self._socket = ssl.wrap_socket(self._socket)
+                # Short reads are not supported in ssl mode. We work around
+                # this by setting the socket non blocking and doing manual
+                # polling/timeout.
+                self._socket.setblocking(False)
```

Comment:
Finally thx a lot for your investigations and contribution
Hope combine new ssl oriented file for mp will be not so hard for you 
Waiting for new PR or next portion of updates  for current PR